src/main/steps/application-within-proceedings/inform-other-parties/content.ts

```diff
@@ -85,7 +87,7 @@ export const form: FormContent = {
   },
   link: {
     classes: 'govuk-!-margin-left-3',
-    href: '/application-within-proceedings/list-of-applications/1',
+    href: applyParms(APPLICATION_WITHIN_PROCEEDINGS_LIST_OF_APPLICATIONS, { pageNumber: '1' }),
```

Comment:
please put this in a util and call it in all content.ts

Reply:
Changed to use constant in utils